### PR TITLE
EPAS - updated the link for data types from v10 to 14 as per DF-309

### PR DESCRIPTION
**Pull request by nidhibhammar**

### CODE REVIEW

File: product_docs/docs/epas/14/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 1]
```diff
@@ -31,9 +31,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by EDB Postgres Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide,* available from EDB at:
-
-[https://www.enterprisedb.com/docs](/epas/latest/epas_compat_reference/)
+For detailed information about the data types supported by EDB Postgres Advanced Server, see the [Database Compatibility for Oracle Developers Reference Guide](/epas/latest/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/14/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:34]:
```suggestion
For detailed information about the data types supported by EDB Postgres Advanced Server, see the [Data types](/epas/latest/epas_compat_reference/02_the_sql_language/02_data_types/).
```

File: product_docs/docs/epas/13/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 2]
```diff
@@ -35,9 +35,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide*, available from EDB at:
-
-[https://www.enterprisedb.com/docs](/epas/latest/epas_compat_reference/)
+For detailed information about the data types supported by Advanced Server, see the [Database Compatibility for Oracle Developers Reference Guide](/epas/13/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/13/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:38]:
```suggestion
For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/13/epas_compat_reference/02_the_sql_language/02_data_types/).
```

Comment (drothery-edb) [inline product_docs/docs/epas/13/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:38]:
```suggestion
For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/13/epas_compat_reference/02_the_sql_language/02_data_types/).
```

Comment (drothery-edb) [inline product_docs/docs/epas/13/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:38]:
```suggestion
For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/13/epas_compat_reference/02_the_sql_language/02_data_types/).
```

File: product_docs/docs/epas/12/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 3]
```diff
@@ -32,9 +32,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide*, available from EnterpriseDB at:
-
-[https://www.enterprisedb.com/docs](/epas/latest/epas_compat_reference/)
+For detailed information about the data types supported by Advanced Server, see the [Database Compatibility for Oracle Developers Reference Guide](/epas/12/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:35]:
```suggestion
For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/12/epas_compat_reference/02_the_sql_language/02_data_types/).
```

File: product_docs/docs/epas/11/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 4]
```diff
@@ -32,9 +32,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide*, available from EnterpriseDB at:
-
-[https://www.enterprisedb.com/docs](/epas/11/epas_compat_reference/)
+For detailed information about the data types supported by Advanced Server, see the [Database Compatibility for Oracle Developers Reference Guide](/epas/11/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/11/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:35]:
```suggestion
For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/11/epas_compat_reference/02_the_sql_language/02_data_types/).
```

File: product_docs/docs/epas/10/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 5]
```diff
@@ -32,9 +32,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide*, available from EnterpriseDB at:
-
-[https://www.enterprisedb.com/docs](/epas/10/epas_compat_reference/)
+For detailed information about the data types supported by Advanced Server, see the [Database Compatibility for Oracle Developers Reference Guide](/epas/10/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/10/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:35]:
```suggestion
For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/10/epas_compat_reference/02_the_sql_language/02_data_types/).
```

**Comment by nidhibhammar** [comment]
@drothery-edb, I have added the article and committed the changes

File: product_docs/docs/epas/12/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 6]
```diff
@@ -32,9 +32,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide*, available from EnterpriseDB at:
-
-[https://www.enterprisedb.com/docs](/epas/latest/epas_compat_reference/)
+For detailed information about the data types supported by Advanced Server, see the [Data types](/epas/12/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:35]:
```suggestion
For detailed information about the data types supported by Advanced Server, see [Data types](/epas/12/epas_compat_reference/02_the_sql_language/02_data_types/).
```

File: product_docs/docs/epas/14/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx [hunk 7]
```diff
@@ -31,9 +31,7 @@ However, certain data types have alternative names referred to as *aliases*, whi
 
 For example, there are fixed length character data types that can be specified as `CHAR` or `CHARACTER`. There are variable length character data types that can be specified as `CHAR VARYING, CHARACTER VARYING, VARCHAR,` or `VARCHAR2`. For integers, there are `BINARY_INTEGER, PLS_INTEGER,` and `INTEGER` data types. For numbers, there are `NUMBER, NUMERIC, DEC,` and `DECIMAL` data types.
 
-For detailed information about the data types supported by EDB Postgres Advanced Server, see the *Database Compatibility for Oracle Developers Reference Guide,* available from EDB at:
-
-[https://www.enterprisedb.com/docs](/epas/latest/epas_compat_reference/)
+For detailed information about the data types supported by EDB Postgres Advanced Server, see the [Data types](/epas/latest/epas_compat_reference/02_the_sql_language/02_data_types/).
```
Comment (drothery-edb) [inline product_docs/docs/epas/14/epas_compat_spl/02_spl_programs/07_subprograms_subprocedures_and_subfunctions/06_overloading_subprograms.mdx:34]:
```suggestion
For detailed information about the data types supported by EDB Postgres Advanced Server, see [Data types](/epas/latest/epas_compat_reference/02_the_sql_language/02_data_types/).
```

**Review by drothery-edb — APPROVED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**